qiskit/quantum_info/analysis/distance.py

```diff
@@ -72,4 +72,4 @@ def hellinger_fidelity(dist_p, dist_q):
 
     dist = np.sqrt(total)/np.sqrt(2)
 
-    return 1-dist
+    return (1-dist**2)**2
```

Comment:
Since majority of the function is computing the Hellinger distance, and the distance has a well defined definition (and wikipedia page) we can link to we should probably just have a function `hellinger_distance(p_dist, q_dist)`.

Reply:
Ok, I can split.